src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -902,6 +903,13 @@ def load_arguments(self, _):
         c.argument('ingress_gateway_type',
                    arg_type=get_enum_type(ingress_gateway_types))
 
+    with self.argument_context('aks mesh enable') as c:
+        c.argument('key_vault_id', required=False, help='The Azure Keyvault id with plugin CA info.', validator=validate_azure_service_mesh_pluginca)
```

Comment:
As you've set the default value (None) for these parameters, no need to set `required=False,`

Reply:
Also you've written the help message in _help.py, may omit here